lib/vtls/schannel.c

```diff
@@ -1574,7 +1594,332 @@ CURLcode Curl_schannel_random(unsigned char *entropy, size_t length)
   return CURLE_OK;
 }
 
-#ifdef _WIN32_WCE
+#ifdef HAVE_EXCLUSIVE_TRUST_MODE
+static CURLcode add_certs_to_store(HCERTSTORE trust_store,
```

Comment:
Since schannel.c is slowly growing in number of lines, I would suggest moving all the custom certificate handling into a separate helper source file, e.g. schannel_cert.c. What do you think, @bagder and @jay?

Reply:
I will echo this , I think the verify stuff could be broken away schannel_verify/cert/whatever.c like how polarssl threadlock stuff went in polarssl_threadlock.c instead

Reply:
Sounds good, I'll attempt that change.